libr/anal/d/types-darwin.sdb.txt

```diff
@@ -0,0 +1,159 @@
+__assert_fail=func
+func.__assert_fail.args=4
+func.__assert_fail.arg.0=const char *,assertion
+func.__assert_fail.arg.1=const char *,file
+func.__assert_fail.arg.2=unsigned int,line
+func.__assert_fail.arg.3=const char *,function
+func.__assert_fail.noreturn=true
+func.__assert_fail.ret=void
+
+_exit=func
+func._exit.args=1
+func._exit.arg.0=int,status
+func._exit.noreturn=true
+func._exit.ret=void
+
+_Exit=func
+func._Exit.args=1
+func._Exit.arg.0=int,status
+func._Exit.noreturn=true
+func._Exit.ret=void
+
+__libc_init=func
+func.__libc_init.args=3
+func.__libc_init.arg.0=int,argc
+func.__libc_init.arg.1=char **,argv
```

Comment:
So is this the right way to define pointer-arguments? Should we use it everywhere? For example, `types-windows.sdb.txt` contains such entries as `argtype*,argname` and `argtype,*argname`.

Reply:
@sivaramaaa would know